dart/test/hub_test.dart

```diff
@@ -114,7 +114,7 @@ void main() {
       expect(capturedEvent.event.contexts.trace, isNotNull);
     });
 
-    test('Expando does not throw when exception type is not supported',
+    test('Expand does not throw when exception type is not supported',
```

Comment:
That's `Expando` that comes from `dart.core`

Reply:
🤦